src/components/Clock/Clock.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './Clock.css';
+import moment from 'moment';
+
+class Clock extends React.Component {
+  state = {
+    months: undefined,
+    days: undefined,
+    hours: undefined,
+    minutes: undefined,
+    seconds: undefined
+  };
+
+  componentDidMount() {
+    this.interval = setInterval(() => {
+      const { timeTillDate, timeFormat } = this.props;
+      const then = moment(timeTillDate, timeFormat);
+      const now = moment();
+      const Clock = moment(then - now);
+      const months = Clock.format('MM');
+      const days = Clock.format('DD');
+      const hours = Clock.format('HH');
+      const minutes = Clock.format('mm');
+      const seconds = Clock.format('ss');
+
+      this.setState({ months, days, hours, minutes, seconds });
+    }, 1000);
+  }
+
+  componentWillUnmount() {
+    if (this.interval) {
+      clearInterval(this.interval);
+    }
+  }
+
+  render() {
+    const { months, days, hours, minutes, seconds } = this.state;
+    const { timeTillDate, timeFormat } = this.props;
+    const then = moment(timeTillDate, timeFormat);
+    const now = moment();
+
+    if (now < then) {
```

Comment:
perfekt!

Reply:
Vent litt. Du ønsker kanskje `now <= then` for å pirke. Vil jo være ett sekund i forskjell tror jeg

Reply:
Hadde <= først men da fikk jeg det ikke til å funke. Fiksa flere steder, men glemte å teste den her på nytt, så bra du sa ifra! Gjorde om nå, og funker fint!